lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,21 @@
+# Get the version of a package in various debian repositories
+package DDG::Spice::DebVersion;
+
+use DDG::Spice;
+
+triggers any => "debian version", "debian versions", "deb ver", "debian ver";
+
+# spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'http://sources.debian.net/api/src/$1/';
+
+spice wrap_string_callback => 1;
+
+handle remainder => sub {
+    my $DebPackage = lc $_;
```

Comment:
We should verify that our debian version is actually a deb version and not a multi-word string or something else we shouldn't bother sending to the API.